index.html

```diff
@@ -111,7 +102,19 @@ <h4>Table of Contents</h4>
 
 					<p>If the table of contents is not located in the <a href="#audio-pep">primary entry page</a>, the manifest SHOULD <a href="#table-of-contents">identify the resource</a> that contains the structure.</p>
 
-					<p>When an Audiobook contains additional resources (i.e. supplemental content) a table of contents SHOULD be included, the table of contents SHOULD include a link to all <a href="#audio-resourcelist">resources</a>, and all links SHOULD refer to <a href="#audio-resourcelist">resources</a> within <a href="#audio-bounds">publication bounds</a>.</p>
+					<p>When an Audiobook contains additional resources (i.e. supplemental content):</p>
```

Comment:
I think there should be reference somewhere to the TOC section (2.9.2.3), as well as the TOC structure (Appendix B) of the pub manifest spec. The section gives the impression that _any_ TOC format could be put there, regardless of what the pub manifest spec defined...